e2e/tests-dfx/bootstrap.bash

```diff
@@ -40,18 +40,3 @@ teardown() {
     assert_command_fail dfx bootstrap --port 8000
     assert_match "Cannot forward API calls to the same bootstrap server"
 }
-
-@test "bootstrap supports http requests" {
```

Comment:
Can we use `dfx_new_frontend` here, and keep this test?  Same for the other removed tests in this PR.